components/Blog/SideNav.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+function SideNav ({ schools, selectedSchool, onSchoolSelected }) {
+  return (
+    <div className='side-nav-blog flex flex-column w-40 self-center pa2'>
```

Comment:
change `w-40` to `w-20`

Reply:
it breaks this one https://github.com/teacherfund/TeacherFund_next/pull/145#discussion_r333707062

Reply:
Just for desktop it's fine. We will worry about responsive styling later.